pkg/scheduler/apis/config/types.go

```diff
@@ -192,8 +192,8 @@ type Plugins struct {
 	// Score is a list of plugins that should be invoked when ranking nodes that have passed the filtering phase.
 	Score *PluginSet
 
-	// Reserve is a list of plugins invoked when reserving a node to run the pod.
-	Reserve *PluginSet
+	// Reservation is a list of plugins invoked when reserving and unreserving a node to run the pod.
```

Comment:
I would prefer that we keep the name `Reserve` to avoid increasing the backwards-incompatible changes from v1alpha2 to v1beta1.
I don't think the name Reserve is deceiving. We have `Score`, which has 2 functions: `Score` and `NormalizeScore`.

WDYT @ahg-g @Huang-Wei ?

Reply:
keeping the name Reserve sgtm.

Reply:
Done.